auction-deploy/auction_deploy/cli.py

```diff
@@ -255,6 +251,62 @@ def print_auction_status(auction_address, jsonrpc):
     click.echo("The closing price is:                   " + str(closing_price))
 
 
+@main.command(short_help="Whitelists addresses for the auction")
+@click.option(
+    "--file",
+    help="Path to the csv file containing the addresses to be whitelisted",
+    type=click.Path(exists=True, dir_okay=False),
+    required=True,
+)
+@click.option(
+    "--auction-address", help="Address of the auction contract", type=str, required=True
+)
+@click.option(
+    "--batch-size",
+    help="Number of addresses to be whitelisted within one transaction",
+    type=click.IntRange(min=1),
+    show_default=True,
+    default=100,
+)
+@keystore_option
+@gas_option
+@gas_price_option
+@nonce_option
+@jsonrpc_option
```

Comment:
add `auto-nonce` option, now that it is merge with master

Reply:
ok